src/amo/reducers/abuse.js

```diff
@@ -156,7 +156,8 @@ type Action =
 
 export default function abuseReducer(
   state: AbuseState = initialState,
-  action: Action,
+  // $FlowIgnore
+  action: Action = {},
```

Comment:
Not sure how best to handle that. This is because `eslint` (rightly) complains that default values should be last. I don't think we care about the `action` in that case...